### PR TITLE
fix: studio 404 not found

### LINKED ISSUE #244: Studio 404 not found
#### Describe the bug

I get this error:
```
$ ./node_modules/.bin/asyncapi start:studio
(node:89108) [MODULE_NOT_FOUND] Error Plugin: @asyncapi/cli: Cannot find module '@oclif/plugin-help/lib/command'
Require stack:
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@fmvilas/oclif-plugin-spaced-commands/lib/hooks/init.js
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@oclif/config/lib/config.js
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@oclif/config/lib/index.js
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@oclif/command/lib/command.js
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@oclif/command/lib/index.js
- /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@asyncapi/cli/bin/run
module: @oclif/config@1.18.3
task: runHook init
plugin: @asyncapi/cli
root: /Users/m/PhpstormProjects/spec/examples/social-media/node_modules/@asyncapi/cli
See more details with DEBUG=*
(Use `node --trace-warnings ...` to show where the warning was created)
Studio is running at http://localhost:3210?liveServer=3210
Watching changes on file asyncapi.yaml
```

but browser opens up and shows message` 404 - The requested path could not be found`

#### How to Reproduce

- installed locally/globally using `npm install  @asyncapi/cli`
- when I got first error with oclif, I added more packages. 

```
{
  "dependencies": {
    "@asyncapi/cli": "^0.14.1",
    "@oclif/command": "^1.8.16",
    "@oclif/plugin-help": "^5.1.11",
    "oclif": "^2.4.5"
  }
}
```          

- run `./node_modules/.bin/asyncapi start:studio`  or `./node_modules/.bin/asyncapi new` 

the guide to create a new file works and I can verify that a new file was created.

macOS Monterey, node - v17.6.0, npm - 8.5.1

#### Expected behavior

It will open a studio in the browser with the studio.

### PR DESCRIPTION
**Pull request by to-kr**
Conditional resolve of Studio Build depending on installation type.

**Related issue(s)**
Resolves #244

### CODE REVIEW

**Review by derberg — CHANGES_REQUESTED** [review]
@to-kr sorry for jumping into review with such delay 😞 

thanks for trying to fix the issue, but I'm not 100% sure it is the best we can do. It is still kinda hacky to have hardcoded paths

could you try to use [`require.resolve`](https://nodejs.org/api/modules.html#requireresolverequest-options) to dynamically check where `@asyncapi/studio` is located?

File: src/models/Studio.ts [hunk 1]
```diff
@@ -44,9 +44,13 @@ export function start(filePath: string, port: number = DEFAULT_PORT): void {
     }
   });
 
+  const studioBuildFolder = existsSync(resolve(__dirname, '../../node_modules/@asyncapi/studio/build')) ?
+    resolve(__dirname, '../../node_modules/@asyncapi/studio/build') : 
+    resolve(__dirname, '../../../../@asyncapi/studio/build');
```
Comment (derberg) [inline src/models/Studio.ts:49]:
```suggestion
  const studioBuildFolder = require.resolve('@asyncapi/studio/build/index.html');
```

this should be universal solution. Works on my side, npm user. 
wdyt?

Comment (derberg) [reply src/models/Studio.ts:49]: @to-kr

Comment (derberg) [reply src/models/Studio.ts:49]: @to-kr do you plan to continue with this one?

Comment (to-kr) [reply src/models/Studio.ts:49]: Sorry, I didn't catch your previous answer. Your solution looks good. Please fix it yourself.

**Comment by derberg** [comment]
closing in favour of https://github.com/asyncapi/cli/pull/489

**Pull request closed without merging**